opentelemetry-otlp/build.rs

```diff
@@ -6,6 +6,7 @@ use protobuf_codegen::Customize;
 
 #[cfg(all(feature = "grpc-sys", not(feature = "tonic")))]
 use protoc_grpcio::compile_grpc_protos;
+use std::env::var;
```

Comment:
Might need to guard it with `#[cfg(all(feature = "grpc-sys", not(feature = "tonic")))]` so that it will be imported if the `grpc-sys` feature is not enabled.